### DISCUSSION TITLE
Welcome to fido2-net-lib Discussions!

### DISCUSSION THREAD

**Discussion by abergs** [passwordless-lib/fido2-net-lib#489] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zacknewman** [comment]
My name is Zack, and I currently only use this library indirectly via Bitwarden. I am excited at how comprehensive this library is. It very well may be the most comprehensive FIDO2 library around. I apologize for the likely "annoying" question, but is there any timeline/roadmap for a `v4.0.0` release? I am excited for the PRF extension that was added back in August. I'm also excited about the prioritization of Ed25519. Presumably once `v4.0.0` is released, Bitwarden will update its dependency on this library which would allow my YubiKey 5C with firmware 5.4.3 to use that algorithm instead of ES256 as currently the web client correctly chooses ES256 since it's before Ed25519 in the `pubKeyCredParams` array.

Keep up the good work.

**Reply by abergs** [reply]
Hey @zacknewman - your question is not annoying :) I work at Bitwarden (and happen to be one of the authors of this library). There's a couple of fixes we want to land before shipping 4.0 stable that will have breaking API signatures, but nothing huge.

At Bitwarden, we're already consuming the Betas for specific features (e.g. our Passwordless.dev developer API) - and hopefully soon we can ship it as part of our vault too.
